src/app/service/logger/console-logger.service.ts

```diff
@@ -23,7 +23,7 @@ export class ConsoleLoggerService implements Logger {
   /* tslint:disable:no-console */
   get info() {
     if (isDebugMode !== 0 && isDebugMode !== 1 && isDebugMode !== 2) {
-      return console.info.bind(console, [user, now]);
+      return console.warn.bind(console, [user, now]);
```

Comment:
why is this change needed? essentially that would mean all logger.info() calls are executed as warnings

Reply:
console info is not allowed as lint is falling

Reply:
done